cli/kata-check_ppc64le.go

```diff
@@ -44,6 +44,12 @@ func archHostCanCreateVMContainer() error {
 // hostIsVMContainerCapable checks to see if the host is theoretically capable
 // of creating a VM container.
 func hostIsVMContainerCapable(details vmContainerCapableDetails) error {
+
+	_, err := getCPUInfo(details.cpuInfoFile)
```

Comment:
I'm not clear why this call is needed here?

Reply:
@jodh-intel : Else, hostIsVMContainerCapable returns "nil" on ppc64le, resulting in testcase " TestCheckCLIFunctionFail" failure as it expects an error. This is silimar to "func genericHostIsVMContainerCapable" from cli/kata-check.go +367

Reply:
Ah right.

lgtm

Reply:
Hmm, so do we need an update for ARM64 I wonder as it uses the same code?

/cc @Pennyzct.

Reply:
Well, not the same code but the same logic in `hostIsVMContainerCapable()`.

Reply:
@jodh-intel thanks for raising up. we will give an update asap😊.